content/06.model-complexity.md

```diff
@@ -1,34 +1,34 @@
-### Manage model complexity while preserving the value of machine learning
+### Augmenting machine learning models with statistical techniques to reduce misinterpretation when applied to rare disease
 
 
-<!-- TODO: stick to one example method of each approach so that we get to describe why we need to use it --->
-
 Machine learning methods are generally accompanied by a few critical assumptions. 
-Assumption 1, the dataset contains equal number of samples for all classes in a dataset (i.e. no imbalance among classes). 
-2. the dataset is complete with adequate samples for all features (lack of sparsity).
-3. there is no ambiguity about the labels for the samples in the dataset (i.e. no label-noise, applicable for supervised learning methods).
+First, ML methods assume that the dataset contains equal number of samples for all classes in a dataset (i.e. no imbalance among classes). 
+Second, the given dataset is complete with adequate samples for all features (lack of sparsity).
+Third, there is no ambiguity about the labels for the samples in the dataset (i.e. no label-noise, applicable for supervised learning methods).
 Rare disease datasets, however, violate many of these assumptions.
 There is generally a high class imbalance due to small number of samples for specific classes (e.g., only a few patients with a particular rare disease in a health records dataset), abundant label-noise due to incomplete understanding of the disease, and in many cases sparsity in data.
```

Comment:
```suggestion
There is generally a high _class imbalance_ due to small number of samples for specific classes (e.g., only a few patients with a particular rare disease in a health records dataset), the data are often _sparse,_ and there may be abundant _label-noise_ due to incomplete understanding of the disease.
```

Reply:
I tried to call out these terms in a way that mirrors the intro to this section.